UIKit_DZ/UIKit_DZ/MenuItemConfigScreen/MenuItemCofigurationViewController.swift

```diff
@@ -3,5 +3,176 @@
+        UIImage(named: "capuchino"),
+        UIImage(named: "latte")
+    ]
+    /// массив с тюплами нашего заказа
+    var orderCoast = [(name: String, coast: Int)]()
+    ///  ссегментед контрол
+    private lazy var segmentControl = UISegmentedControl()
+    /// лейбл - модификация
+    private lazy var modifireLabel = UILabel()
+    /// лейбл - цена
+    private lazy var priceLabel = UILabel()
+    /// кнопка переход на следующий экран
+    private lazy var orderButton = UniversalButton()
+    /// дефоллтная цена кофе
+    private var price = 100
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configurationNavBar()
+        setupUi()
+    }
+
+    // MARK: - Private Methods
+
+    /// настройка навигейшина
+    private func configurationNavBar() {
+        UIBarButtonItem.appearance().tintColor = UIColor.black
+        view.addSubview(topCustomView)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            image: UIImage(systemName: "arrow.backward"),
+            style: .done,
+            target: self,
+            action: nil
+        )
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(systemName: "paperplane.fill"),
+            style: .done,
+            target: self,
+            action: #selector(activityButtonTapped)
+        )
+    }
+
+    /// кладем элементы на экран
+    private func setupUi() {
+        view.backgroundColor = .white
+        topCustomView.addSubview(backButtonBackgroundView)
+        topCustomView.addSubview(itemImage)
+        createLabel(label: modifireLabel, lineX: 16, lineY: 432, title: Constants.modifireLabel)
+        createLabel(label: priceLabel, lineX: 16, lineY: 669, title: "Цѣна - \(price) руб")
+        orderMethod()
+        createSegmendControll()
+        emptyView(title: "Обжарка", lineX: 15)
+        emptyView(title: "Дополнительные /n ингредіенты", lineX: 195)
+    }
+
+    /// создание лейблов
+    private func createLabel(label: UILabel, lineX: Int, lineY: Int, title: String, fontSize: CGFloat = 16) {
+        label.frame = CGRect(x: lineX, y: lineY, width: 345, height: 31)
+        label.text = title
+        label.font = UIFont(name: "HelveticaNeue-Bold", size: fontSize)
+        priceLabel.textAlignment = .right
+        view.addSubview(label)
+    }
+
+    /// создание сегмент контрала
+    private func createSegmendControll() {
+        segmentControl = UISegmentedControl(items: Constants.menuItems)
+        segmentControl.frame = .init(x: 16, y: 368, width: view.frame.width - 32, height: 44)
+        segmentControl.selectedSegmentIndex = 0
+        segmentControl.addTarget(self, action: #selector(selectedItemSegmentControl), for: .valueChanged)
+        view.addSubview(segmentControl)
+    }
+
+    /// создание кнопки
+    private func orderMethod() {
+        orderButton.center.x = view.center.x
+        orderButton.setTitle("Заказать", for: .normal)
```

Comment:
```suggestion
        orderButton.setTitle("Заказать", for: .normal)
```
"Заказать" константа